evm_loader/ERC20/test/test_erc20.py

```diff
@@ -421,8 +436,14 @@ def test_erc20(self):
         assert (erc20.balance(self.ethereum_caller) == 0)
 
         deposit_amount = 1
-        erc20.deposit(self.ethereum_caller, client_acc, self.ethereum_caller, deposit_amount * (10 ** 9),
-                      balance_erc20, token, self.acc.public_key()._key)
+        deposit_trx_data = erc20.deposit(self.ethereum_caller, client_acc, self.ethereum_caller,
+                                         deposit_amount * (10 ** 9), balance_erc20, token,
+                                         self.acc.public_key()._key)
+
+        check_deposit_emulation(self.ethereum_caller.hex(),
```

Comment:
I suggest writing separate tests for check emulation. ERC20 main functional and emulator refer to different functionality. Ideally, tests should show what is not working.

Reply:
Fix. Introduce test_cli_emulate.py